test/lit.cfg.py

```diff
@@ -20,7 +20,7 @@
 
 INCLUDE_INSTALL_PATH = os.environ.get(
     'INCLUDE_INSTALL_PATH',
-    os.path.join(ROOT_PATH, 'build', 'install', 'include'))
+    '$(llvm-kompile --include-dir)')
```

Comment:
The `--include-dir` flag uses the exact same logic as the hard-coded path in this Python code. In `lit`, we know that `llvm-kompile` is on the path (because it's invoked unqualified by `llvm-kompile-testing`). The `INCLUDE_INSTALL_PATH` is only ever used in the context of a lit substitution (i.e. a shell script), so the command substitution here is fine.